magento/magento2
# Fixed: $row() undefined issue while inline edit after delete item from database

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#37525

### Manual testing scenarios (*)

1 Go Cms page list
2 Inline Edit Select the last two or three columns, and make sure the last column selected
![image](https://github.com/magento/magento2/assets/6543606/fc13dfb3-a23e-4cb9-99b0-1b51a73f4178)
3 open another window delete one of the three columns (not the last column) or delete the column in database
![image](https://github.com/magento/magento2/assets/6543606/aad7a4a3-65ef-4881-922f-9630b99c2f49)
4 then save those columns will cause error
![image](https://github.com/magento/magento2/assets/6543606/7764d1dd-b32d-4a33-88bc-db305801eb4f)
5 remove the error column
![image](https://github.com/magento/magento2/assets/6543606/44620374-9a16-4408-8adc-82f754056c1f)
6 save another column (including the last one column), a loading error occurred. still loading
![image](https://github.com/magento/magento2/assets/6543606/04a70d2d-bab8-4a9a-8108-5305aae87d4a)
7 The problem same happened in CMS Block with the same steps
![image](https://github.com/magento/magento2/assets/6543606/78d5d1d6-9aca-4b79-9c34-c469fd4e994b)


### Questions or comments

1. Afrer commit changes continues loading issue has been resolved.

### Contribution checklist (*)
